api/wwwroot/index.html

```diff
@@ -214,6 +224,59 @@ <h2>Get departament</h2>
   "country": null,
   "cities": null
   }
+</code>
+        </div>
+        <div class="overflow-hidden content-section" id="Get-departament">
+          <h2>Get Region</h2>
+          <pre class="json">
+      Get Region
+  </pre>
+          <p>
+            Get the list of all regions in Colombia<br />
+            <code class="higlighted break-word"
+              >/api/{version}/Region</code
+            >
+          </p>
+          <br />
+          <pre><code class="json">
+[
```

Comment:
pls just add one of the demo 
Example:
`{
"id": 1,
"name": "Caribe",
"description": null,
"departments": null
}`

Reply:
ok